debian/control

```diff
@@ -7,6 +7,105 @@ Build-Depends: debhelper (>= 9)
 Standards-Version: 3.9.8
 Homepage: https://github.com/system76/pop-desktop
 
+Package:  pop-container-runtime
```

Comment:
Does this need to be part of the chrony fix commit, or was this an accidental inclusion?

Reply:
Ya total accident. I should have noticed that. Sorry